recipes/automake/all/conanfile.py

```diff
@@ -34,107 +32,91 @@ def configure(self):
-        autotools.make()
-
-    def package(self):
-        self.copy("COPYING*", src=self._source_subfolder, dst="licenses")
-        autotools = self._configure_autotools()
-        autotools.install()
-        tools.rmdir(os.path.join(self._datarootdir, "info"))
-        tools.rmdir(os.path.join(self._datarootdir, "man"))
-        tools.rmdir(os.path.join(self._datarootdir, "doc"))
+    @lru_cache(1)
+    def _autotools(self):
+        autotool = Autotools(self)
+        autotool.configure()
+        autotool.make()
+        return autotool
 
+    def build(self):
+        apply_conandata_patches(self)
         if self.settings.os == "Windows":
-            binpath = os.path.join(self.package_folder, "bin")
-            for filename in os.listdir(binpath):
-                fullpath = os.path.join(binpath, filename)
-                if not os.path.isfile(fullpath):
-                    continue
-                os.rename(fullpath, fullpath + ".exe")
-
-    def package_info(self):
-        self.cpp_info.libdirs = []
-        self.cpp_info.includedirs = []
-
-        bin_path = os.path.join(self.package_folder, "bin")
-        self.output.info("Appending PATH environment variable:: {}".format(bin_path))
-        self.env_info.PATH.append(bin_path)
-
-        bin_ext = ".exe" if self.settings.os == "Windows" else ""
-
-        aclocal = tools.unix_path(os.path.join(self.package_folder, "bin", "aclocal" + bin_ext))
-        self.output.info("Appending ACLOCAL environment variable with: {}".format(aclocal))
-        self.env_info.ACLOCAL.append(aclocal)
-
-        automake_datadir = tools.unix_path(self._datarootdir)
-        self.output.info("Setting AUTOMAKE_DATADIR to {}".format(automake_datadir))
-        self.env_info.AUTOMAKE_DATADIR = automake_datadir
+            # tracing using m4 on Windows returns Windows paths => use cygpath to convert to unix paths
+            replace_in_file(self, path.join(self.source_path, "bin", "aclocal.in"),
+                            "          $map_traced_defs{$arg1} = $file;",
+                            "          $file = `cygpath -u $file`;\n"
+                            "          $file =~ s/^\\s+|\\s+$//g;\n"
+                            "          $map_traced_defs{$arg1} = $file;")
 
-        automake_libdir = tools.unix_path(self._automake_libdir)
-        self.output.info("Setting AUTOMAKE_LIBDIR to {}".format(automake_libdir))
-        self.env_info.AUTOMAKE_LIBDIR = automake_libdir
+        _ = self._autotools()
 
-        automake_perllibdir = tools.unix_path(self._automake_libdir)
-        self.output.info("Setting AUTOMAKE_PERLLIBDIR to {}".format(automake_perllibdir))
-        self.env_info.AUTOMAKE_PERLLIBDIR = automake_perllibdir
+    def package(self):
+        autotools = self._autotools()
+        # KB-H013 we're packaging an application, place everything under bin
+        autotools.install(args=[f"DESTDIR={unix_path(self, path.join(self.package_folder, 'bin'))}"])
```

Comment:
same here

Reply:
Do you mean the specifications of the destdir? This was the only way for me to make sure that the Autotools used the correct Unix path when run on windows with a subsystem.

Also needed because the whole tool is Installed in its entirety in the bindirs.